examples/intellij-plugin/build.gradle.kts

```diff
@@ -4,8 +4,7 @@ plugins {
     id("org.jetbrains.intellij") version "1.1.4"
     java
     kotlin("jvm") version "1.5.31"
-    // __LATEST_COMPOSE_RELEASE_VERSION__
```

Comment:
Please do not remove tag, it's intended to ensure that we never miss an update.

Reply:
Now we only specify folders, and iterate through all files in it. If we would introduce a new pattern (which is rare), we would add it to `replace.sh`

Reply:
Yep, approach with properties seems right and this way tag is not needed.